File: 1939-queries-on-number-of-points-inside-a-circle/1939-queries-on-number-of-points-inside-a-circle.java
class Solution {
    public int[] countPoints(int[][] points, int[][] queries) {
        int[] res = new int[queries.length];
        int i = 0;
        for (int[] q : queries) {
            int count = 0;
            for (int[] p : points) {
                if (Math.pow(p[0] - q[0],2) + Math.pow(p[1] - q[1],2) <= Math.pow(q[2],2)) {
                    count++;
                }
            }
            res[i] = count;
            i++;
        }
        return res;
    }
}

File: 1939-queries-on-number-of-points-inside-a-circle/1939-queries-on-number-of-points-inside-a-circle.py
class Solution:
    def countPoints(self, points: List[List[int]], queries: List[List[int]]) -> List[int]:
        res = [0]*len(queries)
        i = 0
        for x,y,r in queries:
            count = 0
            for x1,y1 in points:
                if (x1 - x)**2 + (y1 - y)**2 <= r**2:
                    count += 1
            res[i] = count
            i += 1
        return res